src/components/InternalDropdown/index.js

```diff
@@ -416,7 +417,13 @@ const InternalDropdown = forwardRef((props, reference) => {
                         </Provider>
                     </Content>
                 </Ul>
-                <RenderIf isTrue={showEmptyMessage}>
+                <RenderIf isTrue={showEmptyMessage && EmptyComponent}>
+                    <EmptyComponent
+                        searchValue={searchValue}
+                        hasTimeout={!!searchTimeout.current}
```

Comment:
here only pass searchValue, for now we don't have use case to use hasTimeout also it is not a meaningful prop name, then for now only searchValue